go.mod

```diff
@@ -19,7 +19,7 @@ require (
 	github.com/google/uuid v1.3.0
 	github.com/hashicorp/go-cty-funcs v0.0.0-20200930094925-2721b1e36840
 	github.com/hashicorp/hcl/v2 v2.8.2
-	github.com/moby/buildkit v0.11.0-rc3.0.20230330090027-8b7bcb900d3c
```

Comment:
Dependabot is confused with Go module pseudo-ver here. `v0.11.0-rc3.0.20230330090027-8b7bcb900d3c` is pseudo-ver from master branch (https://github.com/docker/buildx/pull/1703) which already fixes the CVE: https://github.com/moby/buildkit/commit/75123c696506bdbca1ed69906479e200f1b62604